backend/src/index.ts

```diff
@@ -26,32 +80,42 @@ app.use(
     },
     secret: process.env.SESSION_SECRET as string,
     saveUninitialized: false,
-    resave: true
+    resave: true,
   }),
 );
 
-app.post('/logout', async (req, res) => {
+app.post("/session/validate", async (req: Request, res: Response) => {
```

Comment:
is there a reason why this needs to be an endpoint - can this just be a function that gets called at the start of each endpoint that requires a valid session

Reply:
I was thinking that the frontend needs to change the navbar e.g login / logout based on if the person is logged in, so I was gonna call this each time to change stuff like navbar / profile dropdown. 🔒

Reply:
actually your right i have a user end point for that - 😔 ill remove it